viewer/src/views/threejs/cad/RootSectorNode.ts

```diff
@@ -60,10 +71,18 @@ export class RootSectorNode extends SectorNode {
           return new THREE.Group();
         }
         case LevelOfDetail.Simple: {
-          return consumeSectorSimple(id, data as SectorQuads, metadata, this.materials);
+          const simpleData = data as SectorQuads;
+          this._nodeIdToTreeIndexMap = new Map([...this._nodeIdToTreeIndexMap, ...simpleData.nodeIdToTreeIndexMap]);
+          this._treeIndexToNodeIdMap = new Map([...this._treeIndexToNodeIdMap, ...simpleData.treeIndexToNodeIdMap]);
```

Comment:
Isn't this a very slow way of updating the map? Will do a complete remake every time a sector is loaded?

Reply:
I agree. Would a for-loop be better?

Reply:
Would probably be fastest